requirements.txt

```diff
@@ -1,10 +1,16 @@
-Django==2.1.2
-djangorestframework==3.8.2
-JPype1==0.6.3
-pytz==2018.5
+Django==2.2.14
+jpype1==1.0.1
+numpy==1.19.0
```

Comment:
You should likely upgrade to JPype 1.2 as 1.0 and 1.1 both have a memory leak in strings.